src/Tree/Strategy/ORM/Nested.php

```diff
@@ -647,7 +647,7 @@ public function shiftRangeRL(EntityManagerInterface $em, $class, $first, $last,
         $sign = ($delta >= 0) ? ' + ' : ' - ';
         $absDelta = abs($delta);
         $levelSign = ($levelDelta >= 0) ? ' + ' : ' - ';
-        $absLevelDelta = abs($levelDelta);
+        $absLevelDelta = null !== $levelDelta ? abs($levelDelta) : 0;
```

Comment:
As the type hint for the `$levelDelta` argument only allows `int`, I think we should:
* change its default value to `0`;
* add a deprecation message if other type is passed and cast it to `int` or throw a `\TypeError` error (this is a BC break).